File: src_csp/week0/number_swap.py
from src_csp.util.option import Option


class NumberSwap(Option):
    def __init__(self):
        super().__init__()
        self.name = "Number Swap"

    def tester(self):
        print(self.swap(1, 2))
        print(self.swap(60, 30))
        print(self.swap(9, -11))

    def swap(self, a, b):
        if a > b:
            temp = a
            a = b
            b = temp
        return a, b

File: src/csa/week0/NumberSwap.java
package csa.week0;
import csa.util.Option;

public class NumberSwap extends Option {

    public NumberSwap() {
        optionName = "Number Swap";
    }

    public void tester() {
        System.out.println(swap(1, 2));
        System.out.println(swap(60, 30));
        System.out.println(swap(9, -11));
    }

    private String swap(int a, int b) {
        if (a > b) {
            int temp = a;
            a = b;
            b = temp;
        }
        return (a + " " + b);
    }

}
